### DISCUSSION TITLE
"Position" in config.ini

### DISCUSSION THREAD

**Discussion by ar-wi-88** [fabian-lauer/dbus-shelly-3em-smartmeter#34] [General]
Hello Fabian, 

in the config.ini there is a parameter to set for installation position of the meter. But in the python script it is fixed set to "0".  Maybe you forgot something? :-) 

Thanks for your work!

**Comment by fabian-lauer** [comment]
Hi @ar-wi-88 ,

you are right - did some little changes to the code - included now, working on my test-system.
https://github.com/fabian-lauer/dbus-shelly-3em-smartmeter/commit/441334403e6abcdd8c488b1b6d1c8f0c4cdfdd40

Fabian
